components/admin/users/user-form.tsx

```
@@ -274,13 +282,13 @@ export default function UserForm({ user, onSuccess, isSubmitting = false, onCanc
           <label className="block text-sm font-medium mb-2">{t('USERNAME')} *</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)